src/V3/ResourceNames.cs

```diff
@@ -419,7 +419,7 @@ public static string ParentalStatusView(long customerId, long adGroupId, long cr
             return $"customers/{customerId}/parentalStatusViews/{adGroupId}~{criterionId}";
         }
 
-        public static string PaymentsAccount(long customerId, long paymentsAccountId)
+        public static string PaymentsAccount(long customerId, string paymentsAccountId)
```

Comment:
Please wait for [v4](https://ads-developers.googleblog.com/2020/07/announcing-v4-of-google-ads-api-beta.html) to be released for this client library as you will likely need to update the one file for this other version as well.

Reply:
It's already done on GoB.

Reply:
Ack